apps/balancer-tools/src/app/apr/(utils)/calculateApr.ts

```diff
@@ -166,10 +166,6 @@ async function getFeeAprForDateRange(
   });
 
   if (res.poolSnapshots.length === 0) {
-    // eslint-disable-next-line no-console
-    console.warn(
-      `No data found for pool ${poolId}(${network}) in range ${from} - ${to}`,
-    );
```

Comment:
were we still hitting this many times?

Reply:
This just made me realize this is a similar error case from the below, right? Why don't we treat them the same? I think this might lead to inconsistent results. Did we get to the root cause of why some snapshots are missing? Finding that out is why these logs have been posted here.

Reply:
They are the same error, that's why I removed them from the console warn. We were warned this 3x which was spamming the console and making it harder to debug other things xD 

But the root cause of missing snapshots is not an error, is just how the snapshot works... If no swap was made, then there is no snapshot for that day. Here is an example I looked at yesterday: 

The query is empty, and that is true on balancer FE too

https://app.balancer.fi/#/ethereum/pool/0x6ae5a7857aad26722cb02cc168e38c52e0e4e45d0000000000000000000005dd
[Query on the subgraph](https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-v2/graphql?query=++query+poolSnapshotInRange($poolId:+String!,+$timestamp:+%5BInt!%5D)+%7B%0A++++poolSnapshots(%0A++++++where:+%7B+pool_in:+%5B$poolId%5D,+timestamp_in:+$timestamp+%7D%0A++++++orderBy:+timestamp%0A++++++orderDirection:+asc%0A++++)+%7B%0A++++++pool+%7B%0A++++++++id%0A++++++++address%0A++++++++name%0A++++++++poolType%0A++++++++symbol%0A++++++++tokens+%7B%0A++++++++++symbol%0A++++++++++balance%0A++++++++%7D%0A++++++%7D%0A++++++amounts%0A++++++totalShares%0A++++++swapVolume%0A++++++protocolFee%0A++++++swapFees%0A++++++liquidity%0A++++++swapsCount%0A++++++holdersCount%0A++++++timestamp%0A++++%7D%0A++%7D)

Reply:
so if that's the case, the volume should be 0 for that day, and the liquidity the same from the previous day, right?

Reply:
Yeah, that's why we do the [retry](https://github.com/bleu-fi/balancer-tools/blob/6dbf917b357275f899c8d5b3ba01a6cb7471e6f9/apps/balancer-tools/src/app/apr/(utils)/calculatePoolStats.ts#L51). We missed the fact that the volume should be 0 hehehhehe thanks! (Extra question: I was thinking yesterday If it was possible to get this from the last cached value instead of querying it again but not sure how this would behave when we have multiple days not cached)

But the code we are commenting on right now is the feeApr and collected Fee, if there is no snapshot they should be indeed 0